app/models/services/index.ts

```diff
@@ -142,7 +142,13 @@ function isCollectionMatch(
   operation: DataOperation,
   serviceConfig: ServiceConfig<unknown>,
 ): boolean {
-  return operation.sources.every((source) => serviceConfig.collections.includes(source.collection));
+  return operation.sources.every((source) => {
+    let matchedCollection;
+    if ( serviceConfig.collections.includes(source.collection) ) return true;
+    else if ( matchedCollection = serviceConfig.collections.find(x => x[source.collection]) ) {
```

Comment:
nit: you don't really need the `else`s here since you have returns inside your `if` blocks.

Reply:
Sorry James if I missed something, but I also need to return the expected boolean in `else`s when the condition inside `if` is not true, right?